x-pack/plugins/fleet/cypress/e2e/privileges_integrations_automatic_import_license.cy.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login, loginAndWaitForPage, logout } from '../tasks/login';
+import {
+  CREATE_INTEGRATION_LANDING_PAGE,
+  LICENSE_PAYWALL_CARD,
+} from '../screens/integrations_automatic_import';
+
+describe('User with basic license should hit License Paywall', () => {
+  beforeEach(() => {
+    login();
+    cy.intercept('/api/licensing/info', {
+      license: {
+        uid: 'someId',
+        type: 'basic',
+        mode: 'basic',
+        expiryDateInMillis: 1731226793474,
+        status: 'active',
+      },
+      signature: 'someIdAgain',
+    });
+  });
+  afterEach(() => {
+    logout();
+  });
+
+  it('Create Integration is not accessible when user is basic', () => {
+    cy.request('/api/licensing/info');
```

Comment:
why this request is needed?

Reply:
Right it's not needed. .Fixed in [4217fe2](https://github.com/elastic/kibana/pull/195848/commits/4217fe256f50608d0d8a2dd79507ddc4e3248355)